src/classes/Room.js

```diff
@@ -0,0 +1,12 @@
+class Room {
+  constructor(roomInfo) {
+    this.number = roomInfo.number;
+    this.roomType = roomInfo.roomType;
+    this.bidet = roomInfo.bidet;
```

Comment:
Since `this.bidet` will have a boolean value, this might also work as `this.hasBidet`.